cmd/zpool/zpool_main.c

```diff
@@ -7637,12 +7640,15 @@ status_callback(zpool_handle_t *zhp, void *data)
 			    "contain an on-disk incompatibility\n\twhich "
 			    "needs to be corrected.\n"));
 			(void) printf(gettext("action: To correct the issue "
-			    "enable the bookmark_v2 feature, backup\n\tany "
-			    "existing encrypted datasets to new encrypted "
-			    "datasets,\n\tand destroy the old ones. If this "
-			    "pool does not contain any\n\tencrypted "
+			    "enable the bookmark_v2 feature	Destroy all "
```

Comment:
Typo on this line? It looks like there is a tab between feature and Destroy, with no period.

Reply:
fixed